.github/workflows/compliance.yml

```diff
@@ -50,8 +50,7 @@ jobs:
         # debug
         git log  --pretty=oneline | head -n 10
         west init -l .
-        west update
-        west zephyr-export
+        west update --narrow -o=--depth=1
```

Comment:
I'm afraid this will make the oss-history check fail, because it looks for a merge-base between the current manifest-rev and upstream. This requires having the history.

Reply:
I see. Is the oss-history check part of compliance? Isn't that done in a separate workflow? namely the "OSS history check" which passed for this PR?

Reply:
Ah sorry, that one has its own cloning of the repos.